lessons/Algebra2/exploring-logarithmic-relationships/langs/en-us/index.adoc

```diff
@@ -0,0 +1,163 @@
+
+| Key Points For The Facilitator
+|
+* TBD
+|===
+
+== Looking for Patterns  @duration{20 minutes}
+
+=== Overview
+Students explore the @online-exercise{https://code.pyret.org/editor#share=1SGzPWywGy63-DaqOVTy39-497lcWaF4B, Countries of the World Starter File}, and find a relationship between wealth and median life expectancy. They also learn how to build a new column for a table in Pyret.
+
+=== Launch
+
+[.lesson-instruction]
+--
+@right{@image{images/questions.png, 250}}
+
+- Do people who make more money generally live longer?
+- Open the @online-exercise{https://code.pyret.org/editor#share=1SGzPWywGy63-DaqOVTy39-497lcWaF4B, Countries of the World Starter File}
+- From the File menu, select "Save a Copy", and click "Run".
+- Working in pairs or small groups, complete questions 1-3 on @printable-exercise{exploring-countries.adoc}.
+--
+
+=== Investigate
+
+Discuss students' findings. Pay special attention to the @vocab{form} that students use to describe the pattern in the scatter plot. Is it linear, or something else? Is it strong or weak? If there's some disagreement among the students, that's a good thing!
+
+[.lesson-instruction]
+- Working in pairs or small groups, complete the rest of @printable-exercise{exploring-countries.adoc}.
+
+=== Synthesize
+
+- Have students discuss their findings. What's the best model they came up with?
+- Have students share the sketches they drew. What kinds of lines or curves did they use?
+
+
+== Making Predictions with Logarithmic Models @duration{20 minutes}
+
+=== Overview
+Students try to fit different models to a logarithmic relationship, discovering their limitations. They then fit a logarithmic model to the data, and then try transforming that data using log functions.
+
+=== Launch
+
+@center{@image{images/wealth-v-life-linear.png}}
+
+There definitely appears to be a relationship here, but it's not linear. Using `lr-plot` only shows a moderate correlation, because the straight line doesn't really fit the curve. The shape of this starts out rising fast on the left, before tapering off as `pc-gdp` increases to the right. What functions behave like this?
+
+=== Investigate
+
+The shape of this relationship is _logarithmic_, meaning it can be modeled using a logarithmic function:
+
+@center{   @math{f(x) = b \times log(x) + c}    }
+
+We need to find @math{b} and @math{c}, such that the model fits the data as closely as possible.
+
+Pyret has a few functions that may come in handy here:
+
+- @show{(contract "num-log" '("Number") "Number" "Consumes a number, and produces its natural log")}
+- @show{(contract "num-log-base" '("Number" "Number") "Number" "Consumes a number and a base, and produces the log of that number with the base")}
+- @show{(contract "num-expt" '("Number" "Number") "Number" "Consumes a number and a power, and produces that number raised to the power")}
+
```

Comment:
Wondering if it would be helpful to also include the mathematical formula for each of these. I think it could also help students to see these laid out in a table, with the math expression on top (some images below to show what I'm thinking).
![image](https://user-images.githubusercontent.com/87144723/222742216-efbe6d47-55f0-46f6-bfd9-522c27aa19ca.png)
![image](https://user-images.githubusercontent.com/87144723/222742750-76460dec-2bfe-4a06-830c-cca3a733271d.png)

would suggest the same revision on model-wealth-v-lifespan

Reply:
Done!